src/fights.ts

```diff
@@ -415,18 +416,42 @@ export function dailyFights(): void {
 
           const chip = $item`Pocket Professor memory chip`;
           const jacks = $item`box of Familiar Jacks`;
-          useFamiliar($familiar`Pocket Professor`);
-          if (!have(chip)) {
-            if (mallPrice(jacks) < mallPrice(chip)) {
-              retrieveItem(jacks);
-              use(jacks);
-            } else {
-              retrieveItem(chip);
+          useFamiliar(
+            have($familiar`Pocket Professor`)
+              ? $familiar`Pocket Professor`
+              : $familiar`Comma Chameleon`
+          );
+          if (myFamiliar() === $familiar`Pocket Professor`) {
+            if (!have(chip)) {
+              if (mallPrice(jacks) < mallPrice(chip)) {
+                retrieveItem(jacks);
+                use(jacks);
+              } else {
+                retrieveItem(chip);
+              }
+            }
+          } else {
+            if (get("commaFamiliar") !== $familiar`Pocket Professor`) {
+              if (!have(chip)) {
+                acquire(
+                  1,
+                  chip,
+                  EMBEZZLER_MULTIPLIER() *
+                    get("valueOfAdventure") *
+                    Math.min(40, pocketProfessorLectures() - get("_pocketProfessorLectures"))
+                );
+              }
+              visitUrl(`inv_equip.php?which=2&action=equip&whichitem=${toInt(chip)}&pwd`);
```

Comment:
need to make the chameleon your familiar before calling this, or equip straight to the terrarium